Sources/Platform.swift

```diff
@@ -27,8 +27,8 @@
     public typealias BONFontDescriptor = UIFontDescriptor
     public typealias BONSymbolicTraits = UIFontDescriptor.SymbolicTraits
     let BONFontDescriptorFeatureSettingsAttribute = UIFontDescriptor.AttributeName.featureSettings
-    let BONFontFeatureTypeIdentifierKey = UIFontDescriptor.FeatureKey.featureIdentifier
-    let BONFontFeatureSelectorIdentifierKey = UIFontDescriptor.FeatureKey.typeIdentifier
+    let BONFontFeatureTypeIdentifierKey = UIFontDescriptor.FeatureKey.type
+    let BONFontFeatureSelectorIdentifierKey = UIFontDescriptor.FeatureKey.selector
```

Comment:
These have been deprecated since iOS 8, but we somehow didn't get compile-time warnings, but in this release they must have finally removed them because we get run-time crashes due to failed dynamic symbol loading or something like that. Migrated them to the iOS 9+ variants.

Reply:
There is a radar for this in the release notes: https://developer.apple.com/documentation/xcode-release-notes/xcode-13-beta-release-notes

> Apps that contain Swift code referencing featureIdentifier or typeIdentifier fail to launch on earlier OS releases. (79090498)

Seems like they might be fixing it at some point.